fetch/cosmwasm_contract/src/contract.rs

```diff
@@ -58,14 +63,24 @@ pub fn init<S: Storage, A: Api, Q: Querier>(
         admin: env.message.sender.clone(),
         relayer: env.message.sender.clone(),
         denom: env.message.sent_funds[0].denom.clone(), // TMP(LR)
-        contract_addr_human,                            // optimization
+        contract_addr_human,                            // optimization FIXME(LR) not needed any more (version 0.10.0)
     };
 
     config(&mut deps.storage).save(&state)?;
 
     Ok(InitResponse::default())
 }
 
+/* ***************************************************
+ * **************    Access Control      *************
+ * ***************************************************/
+
+ pub const DEFAULT_ADMIN_ROLE : u64 = 0;
+ pub const RELAYER_ROLE : u64 = 10; // FIXME(LR) arbitrary values, use bytes instead?
+ pub const DELEGATE_ROLE: u64 = 100; 
```

Comment:
Arbitrary values are alright, I would just use values incremeted by 1, e.g. 0, 1, 2, ... besides other things, it is also visually clearer to track.